nitro-container-image/Dockerfile

```diff
@@ -0,0 +1,53 @@
+# docker image for developing and testing Veracruz on AWS Nitro Enclaves
+#
+# AUTHORS
+#
+# The Veracruz Development Team.
+#
+# COPYRIGHT
+#
+# See the `LICENSE.markdown` file in the Veracruz root directory for licensing
+# and copyright information.
+#
+# NOTE: We try to follow the guide in https://docs.docker.com/develop/develop-images/dockerfile_best-practices/
+#       Each RUN contains a bundle of steps, which reduces the cache.
+
+FROM ubuntu:18.04
+
+ARG USER=root
+ARG UID=0
+ENV DEBIAN_FRONTEND noninteractive
+# Use bash as the default
+SHELL ["/bin/bash", "-c"]
+
+RUN apt-get update && apt-get install --no-install-recommends -y \
+    curl && \
+    apt-get clean 
+#    python-setuptools \
+#    python3-setuptools \
+#    python3 \
+#    python3-pip \
+#    musl-tools \
+#    ca-certificates \
+#    lxc \
+#    iptables \
+#    jq \
+#    iproute2
```

Comment:
unused comments?

Reply:
These were not required and for testing they were commented out. No need to keep these